test/integration/pack/test_pack.py

```diff
@@ -696,6 +699,85 @@ def test_project_without_stateboard(cartridge_cmd, project_without_dependencies,
     assert '{}-stateboard.service'.format(project.name) not in systemd_files
 
 
+@pytest.mark.parametrize('build', ['docker', 'local'])
+@pytest.mark.parametrize('pack_format', ['tgz'])
+def test_pack_with_spec_specified(cartridge_cmd, project_without_dependencies, pack_format, build, tmpdir):
+    project = project_without_dependencies
+
+    version = 'scm-2'
+    rockspec_path = get_rockspec_path(project.path, project.name, version)
+    who_am_i = set_whoami_build_specs(rockspec_path, project.name, version)
+
+    cmd = [
+        cartridge_cmd,
+        "pack", pack_format,
+        project.path,
+        "--spec",
+        rockspec_path,
+        "--verbose",
+    ]
+
+    if build == "docker":
+        cmd.append("--use-docker")
+
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc == 0
+    # tarantoolctl performs build with the oldest version of rockspec files
+    assert who_am_i in output
+
+
+@pytest.mark.parametrize('pack_format', ['tgz'])
+def test_packing_with_rockspec_from_other_dir(cartridge_cmd, project_without_dependencies, pack_format, tmpdir):
+    project = project_without_dependencies
+
+    dir_name = 'some_dir'
+    dir_path = os.path.join(project.path, dir_name)
+    os.mkdir(dir_path)
+
+    version = 'scm-2'
+    rockspec_path = get_rockspec_path(dir_path, project.name, version)
+    who_am_i = set_whoami_build_specs(rockspec_path, project.name, version)
+
+    cmd = [
+        cartridge_cmd,
+        "pack", pack_format,
+        project.path,
+        "--spec",
+        rockspec_path,
+        "--verbose",
+    ]
+
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc == 0
+    # tarantoolctl performs build with the oldest version of rockspec files
+    assert who_am_i in output
+
+
+@pytest.mark.parametrize('pack_format', ['rpm', 'deb', 'tgz', 'docker'])
+def test_pack_with_rockspec_bad_name(cartridge_cmd, project_without_dependencies, pack_format, tmpdir):
+    project = project_without_dependencies
+
+    bad_rockspec_name = "bad_rockspec-scm-1.rockspec"
+    bad_rockspec_path = os.path.join(project.path, bad_rockspec_name)
+    rocks_make_output = "Rockspec %s doesn't exist" % (bad_rockspec_path)
```

Comment:
In some places, you use () to pass a single argument to a formatted string, and in others, you [don't](https://github.com/tarantool/cartridge-cli/blob/61e43d1aab391776f1832630871468edf1a80869/test/integration/build/test_build.py#L197). Maybe it's better to stick to one style